src/js_es6/LayoutManager.js

```diff
@@ -593,10 +609,18 @@ export default class LayoutManager extends EventEmitter {
         this._maximisePlaceholder.remove();
         contentItem.parent.callDownwards('setSize');
         this._maximisedItem = null;
+        contentItem.off( 'beforeItemDestroyed', this._$cleanupBeforeMaximisedItemDestroyed, this );
         contentItem.emit('minimised');
         this.emit('stateChanged');
     }
 
+    _$cleanupBeforeMaximisedItemDestroyed() {
```

Comment:
Out of curiosity (I did some googling and couldn't really find a usage of both `_` and `$`), what's the convention here? I know `_` means private, and `$` is frequently used as an alias for `document.getElementById`; but I'm rather new to js and have never seen the `_$` prefix before. What does this mean?

Thanks so much for stepping up and maintaining this project. I'm definitely going to try and contribute as well, once I'm more familiar with js.

Reply:
underscore usually means private, protected or internal.
In the context of rxJS, $ is used to denote observables, in the context of JQuery it is often used to have jquery instances

As far as I can tell, methods prefixed with $ in golden-layout are flagged for usage with `callDownwards` and `callUpwards`

Reply:
Thanks!